### PR TITLE
Show how an 'extra' section in docker-compose.yml can be used.

### PR DESCRIPTION
**Pull request by tbeadle**
### Proposed changes

Added documentation for change proposed in https://github.com/docker/compose/pull/4461

Signed-off-by: Tommy Beadle <tbeadle@gmail.com>

### Unreleased project version (optional)

compose 1.12 ?? (I'm not totally sure)

### CODE REVIEW

**Comment by mstanleyjones** [comment]
@tbeadle please update the status of this PR when the change is accepted and you know the version it's going in. We don't want to accidentally merge docs changes for a feature that isn't in yet.

**Comment by johndmulhausen** [comment]
It looks like a few of these are already in there? Recently pulled this change up from `vnext-compose` and published it: https://github.com/docker/docker.github.io/pull/1601/files

**Comment by mstanleyjones** [comment]
Please reopen this PR if you want to continue with this work.

File: compose/compose-file.md [hunk 1]
```diff
@@ -1409,6 +1410,29 @@ refer to it within the Compose file:
         external:
           name: actual-name-of-network
 
+## Extra configuration reference
```
Comment (londoncalling) [inline compose/compose-file.md]: @shin- @dnephin below is the one aspect of this PR that never made it in to the current docs. It's the addition of an `Extra configuration reference` topic. Can you all look at this content change, and let me know if I should add this to the [current Compose file reference](https://docs.docker.com/compose/compose-file/)?

Thanks, Vicky

Comment (dnephin) [inline compose/compose-file.md]: @londoncalling the PR which adds this feature has not merged, so no docs changes are required.

**Pull request closed without merging**